server/routes/controllers/forgotPasswordController.js

```diff
@@ -1,12 +1,15 @@
 const sendgrid = require('@sendgrid/mail')
 const ForgotPasswordModel = require('../../database/models/ForgotPassword')
+const UserModel = require('../../database/models/User')
 
 const forgotPasswordController = {
-  _init (emailClient = sendgrid, ForgotPass = ForgotPasswordModel) {
+  _init (emailClient = sendgrid, ForgotPass = ForgotPasswordModel, Users = UserModel) {
```

Comment:
Why have default values for parameters on `_init()`?  Is it just to give those variables friendlier names?

Reply:
To be honest, I just did the same as it was in another's files. @bbmoz , can you explain? I believe it's because of the friendlier names.